randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -1201,7 +1201,11 @@ Extra - room_id: [33]
   * Extra - door_idx: (73,)
   > Beside Pickup
       Any of the following:
-          Screw Attack or Space Jump or Can Freeze Enemies With Any Weapon
+          Screw Attack or Space Jump
+          All of the following:
+              # Get over Kago by freezing it
+              Can Freeze Enemies With Any Weapon
+              Hi-Jump or Stand On Frozen Enemies (Beginner)
```

Comment:
what's the point of HJ here when the below is just HJ

Reply:
Please point me to the logic block that jumps over the kago with HJ only.

Reply:
ignore